comp/otelcol/extension/README.md

```diff
@@ -0,0 +1,57 @@
+# Datadog Extension
+
+The datadog extension component allows inclusion of otel-agent data in the datadog-agent [flare](https://docs.datadoghq.com/agent/troubleshooting/send_a_flare/?tab=agent). A flare can be triggered by the core agent process, or remote-config.
+
+## Extension Configuration
+
+The datadogextension will be added automatically by the [converter component](../converter/README.md). If you opted out of the converter, or you want to change the defaults, you are able to configure the extension as so:
+
+*Collector config:*
+```
+extensions:
+ datadog:
+   port: 7777
+```
+
+*Agent Config:*
+```
+otel-agent:
+ enabled: true
+ flare_port: 7777
+```
+
+The port is the location in which the otel-agent will expose the data required to build the flare. The core agent then fetches the data from this port.
+
+## Data collected by flare
```

Comment:
We may want to add a small section below this one about the additional metadata submitted for fleet automation, which includes stuff like the `otel-agent` version, etc.

Reply:
same as above 👍